pkg/server/register.go

```diff
@@ -105,19 +105,13 @@ func (i *InventoryServer) ServeHTTP(resp http.ResponseWriter, req *http.Request)
 }
 
 func (i *InventoryServer) unauthenticatedResponse(machineRegistration *elm.MachineRegistration, writer io.Writer) error {
-	_, err := writer.Write([]byte("#cloud-config\n"))
```

Comment:
This was never meant to be a cloud-config as described by [cloud-init docs](https://cloudinit.readthedocs.io/en/latest/topics/examples.html) neither a yip config file. I have no idea why this mark was included here at some point.

Reply:
yeah... nice cleanup

Reply:
I'm not 100% sure if there was some other intention behind to push some cloud config at those stages, but it looks good as the relevant parts we have to return are just part of the defined schema